### PR TITLE
Make and implement a custom 404 Error page #152

### LINKED ISSUE #152: Make and implement a custom 404 Error page
## Is your feature request related to a problem? Please describe.
Create a custom 404 error page.

## Describe the solution you'd like!
As of now the website uses the default error page by django. Create a template for 404 error page as such and implement it so that a user is redirected to it whenever a 404 error occurs.

## Describe alternatives you've considered
Feel free to revamp the design if you have better ideas.

## Developer Help
**New design:**
Group 14

### PR DESCRIPTION
**Pull request by Vaibhavsahu2810**
Overview
This PR fixes or fixes part of #152
This PR does the following: Created a error 404 page to resolve the issue #152

Proof that changes are correct

https://github.com/BitByte-TPC/alumni/assets/154520239/9f7d9f74-71e3-4e07-bce4-720f6dee8e7a

This screen recording shows that when the user try to visit any undefined route they see an custom error 404 page.
Thank you.

Fixes #152

### CODE REVIEW

File: AlumniConnect/settings/development.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 SECRET_KEY = "dikj)qxgkhe7v$y7l))d!!nkut&^6q7+x^@ys7c1z!#!&*94r5"
 
-DEBUG = True
+DEBUG = False
```
Comment (SamakshAgarwal1112) [inline AlumniConnect/settings/development.py:5]:
```suggestion
DEBUG = False
```
Why is Debug = False? Make it true again. We'll make it false when and if we want to test.

Comment (SamakshAgarwal1112) [inline AlumniConnect/settings/development.py:5]:
```suggestion
DEBUG = False
```

Comment (SamakshAgarwal1112) [inline AlumniConnect/settings/development.py:5]: Why is Debug = False? Make it true again. We'll make it false when and if we want to test.

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 amqp==5.2.0
 astroid==2.1.0
 async-timeout==4.0.3
-backports.zoneinfo==0.2.1
+backports.zoneinfo;python_version<"3.9"
```
Comment (SamakshAgarwal1112) [inline requirements.txt:4]: Why is this change pushed? Never push changes in requirements.txt unless asked.

**Comment by SamakshAgarwal1112** [comment]
@Vaibhavsahu2810 Try to improve responsive a bit more.

**Comment by Vaibhavsahu2810** [comment]
@SamakshAgarwal1112 successfully implemented all the changes.
Thank you

**Comment by SamakshAgarwal1112** [comment]
@Vaibhavsahu2810 The 404 heading is floating to the right on larger screens. Try to keep it in the middle.

**Comment by Vaibhavsahu2810** [comment]
@SamakshAgarwal1112 successfully improved the responsiveness for larger screens
Thank you
Screenshot 2024-06-10 at 2 57 26 PM

**Pull request merged**